keyboards/3.0/mt-t-k0-47key.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE keyboard3 SYSTEM "../dtd/ldmlKeyboard3.dtd">
-<keyboard3 locale="mt-t-k0-47key" conformsTo="techpreview">
+<keyboard3 xmlns="https://schemas.unicode.org/cldr/45/keyboard3" locale="mt-t-k0-47key" conformsTo="techpreview">
```

Comment:
I've only changed one file, but the others can be changed as well once there's consensus.